spec/spec_helper.rb

```diff
@@ -27,6 +27,19 @@
 
 # We need a fake "ApplicationController" because Gaffe's default controller inherits from it
 class ApplicationController < ActionController::Base
+  if Rails::VERSION::MAJOR < 4
+    def self.before_action(*args)
+      before_filter(*args)
+    end
+  end
```

Comment:
Same thing here, `before_action` should work correctly in Rails 4’s `ActionController::Base`.